src/services/nnas/routes/support.ts

```diff
@@ -125,6 +125,35 @@ router.get('/resend_confirmation', async (request: express.Request, response: ex
 	response.status(200).send('');
 });
 
+/**
+ * [GET]
+ * Replacement for: https://account.nintendo.net/v1/api/support/send_confirmation/pin/:email
+ * Description: Sends a users confirmation email
+ */
+router.get('/send_confirmation/pin/:email', async (request: express.Request, response: express.Response): Promise<void> => {
+	const email = String(request.params.email)

```

Comment:
`String` here is redundant, since it's already a string. Also missing a semicolon